docs/stark-build/NG_CLI_BUILD_CUSTOMIZATIONS.md

```diff
@@ -284,11 +292,7 @@ the following lines in your `<head>` section:
 
 ```html
 <head>
-	...
-	<% if (webpackConfig.htmlElements.headTags) { %>
-	<%= webpackConfig.htmlElements.headTags %>
-	<% } %>
-	...
+	... <% if (webpackConfig.htmlElements.headTags) { %> <%= webpackConfig.htmlElements.headTags %> <% } %> ...
```

Comment:
Can you revert this change? I know this is done automatically by Prettier but it is less readable this way :(

Reply:
@SuperITMan can you revert this formatting? Prettier does it automatically but I think it is less readable this way :(

Reply:
Quickly updated, I accidentally ran prettier again after changing it I think. 😅